### DISCUSSION TITLE
[ UPDATE!!! ] LAST MAN STANDING

### DISCUSSION THREAD

**Discussion by Jeyphr** [nzp-team/nzportable#795] [Map Releases]
### WELCOME TO LAST MAN STANDING!
_1st official uploaded map babey!_

Verse against your friends
in this chaotic single-room
zombie survival challenge.

### [>>> DOWNLOAD LINK <<< ](https://archive.org/download/lastManStanding_v1/Last%20Man%20Standing%20%28v1.5%29.zip)
[J.A.C.K Project Files](https://archive.org/details/lastManStanding_v1)

---
### SCREENIES
<img width="1359" height="767" alt="image" src="https://github.com/user-attachments/assets/91a24eef-0c93-4cc6-850c-d0b25ddf158c" />
<img width="512" height="256" alt="LastManStanding" src="https://github.com/user-attachments/assets/4c2b6d2c-ef63-4366-85ff-072b5ab9677a" />

---
### Update Log
- Added simple optimization techniques
- Removed JTEXT.wad, used -wadinclude
- Updated the sky texture
- Updated ambient track
- Added Developer Credits in intro text and join message

**Comment by zqpvr** [comment]
Where do I put the wad at

**Reply by Jeyphr** [reply]
textures folder

**Comment by doromiert** [comment]
played it with @Jeyphr and @Blade04208, this map is really fun

**Reply by Blade04208** [reply]
Yup, is based map

**Reply by spectrum1-wave** [reply]
how do you load the map?

**Comment by SnugPugUnder** [comment]
I would love for a shadows of evil map. 👍 I don’t know how to make a map on chrome.

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the archive is not equivalent to the gamedir root: https://github.com/nzp-team/nzportable/discussions/159

**Comment by MisaelJama** [comment]
this map doesn't work on psp right
